performance_test/script.mjs

```diff
@@ -0,0 +1,98 @@
+import { execSync } from "child_process";
```

Comment:
let's add more logs to know what's the status, like initialisation, cleaning, computing

Reply:
done.